QtCollider/widgets/QcWebView.cpp

```diff
@@ -228,29 +223,36 @@ void WebView::contextMenuEvent(QContextMenuEvent* event) {
     menu.exec(event->globalPos());
 }
 
-void WebView::keyPressEvent(QKeyEvent* e) {
-    int key = e->key();
-    int mods = e->modifiers();
-
-    if (_interpretSelection
-        && (key == Qt::Key_Enter || (key == Qt::Key_Return && mods & (Qt::ControlModifier | Qt::ShiftModifier)))) {
-        QString selection = selectedText();
-        if (!selection.isEmpty()) {
-            Q_EMIT(interpret(selection));
-            return;
-        }
+// webView's renderer keypresses don't arrive to webView
+// duplicate them
+bool WebView::eventFilter(QObject* obj, QEvent* event) {
+    if (event->type() == QEvent::KeyPress) {
+        QKeyEvent* keyEvent = static_cast<QKeyEvent*>(event);
+        QKeyEvent* newEvent =
+            new QKeyEvent(QEvent::KeyPress, keyEvent->key(), keyEvent->modifiers(), keyEvent->nativeScanCode(),
+                          keyEvent->nativeVirtualKey(), keyEvent->nativeModifiers(), keyEvent->text());
+        QApplication::postEvent(this, newEvent);
```

Comment:
i would recommend this. i'm pretty sure QKeyEvent has a copy ctor.

```suggestion
        // takes ownership of newEvent
        QApplication::postEvent(this, new QKeyEvent(*static_cast<QKeyEvent*>(event))); 
```